collections-csharp/UpdateItemInListCSharp/src/UpdateItemInListCSharp/Models/Library.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace UpdateItemInListCSharp.Models
+{
+    public class Library
+    {
+        public List<Book> Books { get; set; }
+
+        public Library()
+        {
+            Books =
+                [
+                new("J.K. Rowling", "978-0439708180", false, "Harry Potter and the Philosopher's Stone"),
+                new("J.K. Rowling", "978-0439064873", false, "Harry Potter and the Chamber of Secrets"),
+                new("J.K. Rowling", "978-0439139601", false, "Harry Potter and the Goblet of Fire"),
+                new("J.K. Rowling", "978-0439358071", false, "Harry Potter and the Order of the Phoenix"),
+                new("J.K. Rowling", "978-0439785969", false, "Harry Potter and the Half-Blood Prince")
+            ];
+        }
+
+        public Book AddBook(string author, string isbn, bool isCheckedOut, string title)
+        {
+            var book = new Book(author, isbn, isCheckedOut, title); Books.Add(book);
+            return book;
+        }
+
+        public void CheckoutBookFind(string isbn)
```

Comment:
Please rename the method from "CheckoutBookFind" to "CheckoutBookUsingFind". Use the work "Using" before method name, applicable for all the methods in this class